src/types.ts

```diff
@@ -0,0 +1,10 @@
+export interface Community {
+  title: string;
+  url: string;
+  members: number;
+  image: string;
+}
+
+export interface HomeProps {
```

Comment:
Homeprops should not be inside here. It random that its currently the same, home will eventually need more data and have another prop other than Community and then it wont be the same as TrendingProps.